diagnostics.md

```diff
@@ -69,14 +69,14 @@ In the event of a problem, the diagnostic view would help you identify specific
 
 Note the file on `Nov 20, 2020` in the above image appears in <span style="color:red">**red**</span>. The data in this file has been flagged by our diagnostics page as *empty* (failed to pass a minimum size threshold). This could happen if the database was temporily unavailable while building that day.
 
-The ETL pipelines provide a way to rebuild a specific day in the pipeline using the following URL:
+The ETL pipelines provide a way to repair a specific day in the pipeline using the following URL:
 ```
-http://<kubecost-url>:<port>/model/etl/[allocation|assets]/rebuild?window=<RFC3339-start>,<RFC3339-end>&commit=true
+http://<kubecost-url>:<port>/model/etl/[allocation|asset]/repair?window=<RFC3339-start>,<RFC3339-end>
 ```
 
-In order to rebuild the file for the problematic date above (Note it's for _Allocation_), I would navigate to the following in a browser:
+In order to repair the file for the problematic date above (Note it's for _Allocation_), navigate to the following in a browser:
 ```
-http://<kubecost-url>:<port>/model/etl/allocation/rebuild?window=2020-11-20T00:00:00Z,2020-11-21T00:00:00Z&commit=true
+http://<kubecost-url>:<port>/model/etl/allocation/repair?window=2020-11-20T00:00:00Z,2020-11-21T00:00:00Z
```

Comment:
These are the only real changes. Everything else is linting.

Reply:
We need to maintain some of the old stuff for pre 1.82.0 and note it for people running versions less than 1.82.0

Reply:
Do we generally do that inline, or is there some other documentation versioning idea with tagging or otherwise?

Reply:
inline is fine

Reply:
Added a note below